develop/api_guc.rst

```diff
@@ -222,6 +222,19 @@ The default value for this GUC is 'on'.
 Planner Configuration
 ------------------------------------------------
 
+citus.local_table_join_policy (enum)
```

Comment:
We should add a link target above this, like:

```rst
.. _local_table_join_policy:
```

That way Onur can link to it from https://github.com/citusdata/citus_docs/pull/983